tools/ports/contrib/lua.py

```diff
@@ -0,0 +1,51 @@
+# Copyright 2025 The Emscripten Authors.  All rights reserved.
+# Emscripten is available under two separate licenses, the MIT license and the
+# University of Illinois/NCSA Open Source License.  Both these licenses can be
+# found in the LICENSE file.
+
+import os
+
+TAG = '5.4.7'
+HASH = '98c5c8978dfdf867e37e9eb3b3ec83dee92d199243b5119505da83895e33f10d43c841be6a7d3b106daba8a0b2bd25fe099ebff8f87831dcc55c79c78b97d8b8'
+ZIP_URL = f'https://www.lua.org/ftp/lua-{TAG}.tar.gz'
+
+# contrib port information (required)
+URL = 'https://www.lua.org/'
+DESCRIPTION = 'Lua is a powerful, efficient, lightweight, embeddable scripting language'
+LICENSE = 'MIT License'
+
+port_name = 'contrib.lua'
+
+
+def get_lib_name(settings):
+  return f'lib_{port_name}.a'
```

Comment:
Can we call the library just `liblua.a`.   You don't need `get_lib_name` then I think.

Reply:
done